woodwork/tests/utils/test_utils.py

```diff
@@ -326,18 +326,22 @@ def test_to_latlong_float():
         _to_latlong_float([1, 2, 3])
 
 
-def test_is_null_latlong():
-    assert _is_null_latlong(None)
-    assert _is_null_latlong(np.nan)
-    assert _is_null_latlong(pd.NA)
-    assert _is_null_latlong("None")
-    assert _is_null_latlong("nan")
-    assert _is_null_latlong("NaN")
+def _is_nan():
+    assert _is_nan(None)
+    assert _is_nan(np.nan)
+    assert _is_nan(pd.NA)
+    assert _is_nan("None")
+    assert _is_nan("nan")
+    assert _is_nan("NaN")
+    assert _is_nan("<NA>")
+    assert _is_nan("")
+    assert _is_nan("null")
 
-    assert not _is_null_latlong([None, 1, 3])
-    assert not _is_null_latlong("none")
-    assert not _is_null_latlong(0)
-    assert not _is_null_latlong(False)
+    assert not _is_nan([None, 1, 3])
```

Comment:
I like the inclusion of all the falsy values here. Thoughts on also having the empty list `[]` or some other data structure like a set or dict?